crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
```

Comment:
@MaximilianMeister the naming is a bit difficult. We're not searching disks that are unreferenced but for claims that reference disk(name)s that are no longer existing.